core/src/main/java/nl/nn/adapterframework/util/AppConstants.java

```diff
@@ -172,23 +182,23 @@ public String getResolvedProperty(String key) {
 	}
 
 	/**
-	 * Creates a tokenizer from the values of this key.  As a sepearator the "," is used.
+	 * Creates a tokenizer from the values of this key. As a separator the "," is used.
 	 * Uses the {@link #getResolvedProperty(String)} method.
 	 * Can be used to process lists of values.
 	 */
 	public StringTokenizer getTokenizer(String key) {
-	    return new StringTokenizer(getResolvedProperty(key), ",");
+		return new StringTokenizer(getResolvedProperty(key), ",");
 	}
 	/**
-	 * Creates a tokenizer from the values of this key.
+	 * Creates a tokenizer from the values of this key. As a separator the "," is used.
 	 * Uses the {@link #getResolvedProperty(String)} method.
 	 * Can be used to process lists of values.
 	 */
 	public StringTokenizer getTokenizer(String key, String defaults) {
```

Comment:
We zouden kunnen bedenken deze getTokenizer method te hernoemen naar: getTokenizedProperty() of getListProperty

Reply:
Ja hij wordt ook maar op een drietal plekken gebruikt. Ik vind het prima en stem dan voor getTokenizedProperty